pkg/system/rm.go

```diff
@@ -29,7 +29,9 @@ func EnsureRemoveAll(dir string) error {
 	maxRetry := 100
 
 	// Attempt to unmount anything beneath this dir first
-	mount.RecursiveUnmount(dir)
+	if err := mount.RecursiveUnmount(dir); err != nil {
```

Comment:
The comment says "attempt" so I guess we'd rather ignore an error here and still try to remove what we can (and then return an error from `RemoveAll()` if there is any).

Reply:
Fixed to just print debug messages on failures.